### PR TITLE
Clean up some texts for wraparound checks

### PR DESCRIPTION
**Pull request by junyechen1996**
We have conflicting references of "wraparound check results", e.g., "Encoding range-checked wraparound check results", which are the bits of the range-checked result in wraparound check, and we also have "wraparound check results", that we typically mean the dot products from running wraparound tests.

How about we use "check" on the entire wraparound "check" section, i.e., it is checking for wraparound, similar to L2-norm "check". And we also have the quadratic "check" and success count "check" in wraparound checks.
For the dot product computation, we will refer to it as wraparound "test". I think we already did this in a couple of places, but just weren't consistent about it.

### CODE REVIEW

**Comment by junyechen1996** [comment]
@cjpatton If you like this idea, I can make a pass over the variables (mainly `num_wr_checks` -> `num_wr_tests`)  and reference implementations.

**Comment by cjpatton** [comment]
> We have conflicting references of "wraparound check results", e.g., "Encoding range-checked wraparound check results", which are the bits of the range-checked result in wraparound check, and we also have "wraparound check results", that we typically mean the dot products from running wraparound tests.
> 
> How about we use "check" on the entire wraparound "check" section, i.e., it is checking for wraparound, similar to L2-norm "check". And we also have the quadratic "check" and success count "check" in wraparound checks. For the dot product computation, we will refer to it as wraparound "test". I think we already did this in a couple of places, but just weren't consistent about it.

I'd prefer to use one term: either "check" or "test". Mixing them would be confusing.

Instead, I think our goal should be to make sure the distinction is clear in context: one is encoded in the in the input (shares), the other is computed from the input (shares).

**Comment by junyechen1996** [comment]
> Instead, I think our goal should be to make sure the distinction is clear in context: one is encoded in the in the input (shares), the other is computed from the input (shares).

Got it. I reworked this PR and tried to clarify the distinction between them.

File: draft-chen-cfrg-vdaf-pine.md [hunk 1]
```diff
@@ -384,17 +384,17 @@ gradient and the random vector derived on their own.
 We define a function `PineValid.encode_wr_checks(self,
 encoded_gradient: list[Field], wr_joint_rand_xof: Xof) ->
 tuple[list[Field], list[Field]]` that implements this encoding step. It returns
-a tuple of range-checked, wraparound check results, and the wraparound check
-results (i.e., the dot products) from {{run-wr-check}}.
+a tuple of range-checked, wraparound check results that will be sent to the
```
Comment (cjpatton) [inline draft-chen-cfrg-vdaf-pine.md:387]:
```suggestion
the tuple of range-checked, wraparound check results that will be sent to the
```

**Review by cjpatton — APPROVED** [review]

**Pull request merged**